src/main/java/de/tum/cit/aet/artemis/lecture/service/SlideSplitterService.java

```diff
@@ -89,10 +131,13 @@ public void splitAttachmentUnitIntoSingleSlides(PDDocument document, AttachmentU
                 MultipartFile slideFile = fileService.convertByteArrayToMultipart(filename, ".png", imageInByte);
                 Path savePath = fileService.saveFile(slideFile, FilePathService.getAttachmentUnitFilePath().resolve(attachmentUnit.getId().toString()).resolve("slide")
                         .resolve(String.valueOf(slideNumber)).resolve(filename));
+
                 Slide slideEntity = new Slide();
                 slideEntity.setSlideImagePath(FilePathService.publicPathForActualPathOrThrow(savePath, (long) slideNumber).toString());
                 slideEntity.setSlideNumber(slideNumber);
                 slideEntity.setAttachmentUnit(attachmentUnit);
+                slideEntity.setHidden(null);
+                slideEntity.setExercise(null);
```

Comment:
is it expected that after splitting the slide
1. slides are not referenced by the exercise anymore and
2. there is no hidden date, i.e. students could potentially see those (?)